core/templates/domain/objects/objects-domain.constants.ts

```diff
@@ -64,4 +64,20 @@ export class ObjectsDomainConstants {
       base_unit: '0.01 rupee'
     }
   };
+  public static RATIO_PARSING_ERRORS = {
+    INVALID_COLONS:
+      'Your answer has two colons (:) next to each other.',
+    INVALID_CHARS:
+      'Please write a ratio that consists of digits separated by colons' +
+      '(e.g. 1:2 or 1:2:3).',
+    INVALID_FORMAT:
+      'Please enter a valid ratio (e.g. 1:2 or 1:2:3).',
+    NON_INTEGER_ELEMENTS:
+      'For this question, each element in your ratio should be a whole ' +
+      'number (not a fraction or a decimal).',
+    INCLUDES_ZERO:
+      'Ratio cannot have 0 as a element.',
```

Comment:
```suggestion
      'Ratios cannot have 0 as a element.',
```

Otherwise this reads slightly like broken English.

Reply:
done